src/uu/chmod/src/chmod.rs

```diff
@@ -37,13 +37,54 @@ mod options {
 
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
-    let mut args = args.collect_lossy();
+    let args = args.collect_lossy();
```

Comment:
We should open an issues for the `collect_lossy`. `chmod` should allow non-utf-8 inputs.

Reply:
I can do that. I will extract my current iteration into an own function and return Paths from there for obvious potential paths (like non utf8 args)

Reply:
Let's do it separately from this PR. It's just something that I noted for later :)

Reply:
I added a couple of FIXMEs for it

Reply:
And I am not sure why the github-actions bot is not posting a "no longer failing".

Reply:
Hahaha you've already solved 80% of the issue now :)

> I can create a (for now failing, but actually working) non-utf8/utf16 test to cover it :)

Let's add it once we fix it. It's unrelated to this PR anyway.

> And I am not sure why the github-actions bot is not posting a "no longer failing".

Yeah something seems broken:

```
Warning: Skipping test failure comparison; no prior reference test logs are available.
```